### PR TITLE
Adds previously dynamic properties for PHP 8.2 compatibility

### PR DESCRIPTION
**Pull request by gsteel**
At the moment, `laminas-filter` needs this for it's CI to pass. See 

https://github.com/laminas/laminas-filter/pull/90

and one of the run failures here

https://github.com/laminas/laminas-filter/actions/runs/3717446853/jobs/6304805223

### CODE REVIEW

**Comment by Ocramius** [comment]
CI errors seem to be related with OpenSSL no longer supporting certain operations: unsure how to proceed here :thinking:

File: src/Symmetric/Openssl.php [hunk 1]
```diff
@@ -167,6 +167,19 @@ class Openssl implements SymmetricInterface
...
+    /**
+     * Supported algo list
+     *
+     * This property was previously dynamic. It should be made private in the next major, therefore marked internal.
+     *
+     * @internal
+     *
+     * @link self::getSupportedAlgorithms()
+     *
+     * @var list<string>|null
```
Comment (Ocramius) [inline src/Symmetric/Openssl.php:179]: `|null` conflicts with the typed property declaration

Comment (gsteel) [reply src/Symmetric/Openssl.php:179]: 👍 removed the type for now. There's no psalm in this repo either.

**Comment by gsteel** [comment]
Maybe it'd be easier to drop the encryption adapters from `laminas-filter` and release a major?

**Comment by Ocramius** [comment]
@gsteel how about:

 1. deprecating them
 2. removing their tests too

That would give some space for moving forward, whilst preventing a major (for now)

**Comment by gsteel** [comment]
Closing this PR because the problem with `laminas-filter` is now solved

**Pull request closed without merging**